client/src/app/components/edit-profile/edit-profile.component.scss

```diff
@@ -0,0 +1,85 @@
+@import "../../../styles.scss";
+.wrapper {
+    max-width: 575px;
+    margin-left: auto;
+    margin-right: auto;
+    .cover-picture {
```

Comment:
I know this isn't in figma but I think it would look good if you add a 5px border radius to the background cover

Reply:
White / black?

Reply:
it's border radius!